index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+    <title>SAMI ULLAH TUFAIL </title>
+</head>
+<body>
+  <header id="toolbar">
+    <a id="logo" class="margin-reset" href="">Sami Ullah</a>
+    <a id="nav-button"><img src="ASSETS /Union.png" alt="navigation bar button"></a>
+    <nav id="nav-list" class="hide">
+      <a id="close-navlist" class="hide-desk"><img src="ASSETS /Icon.png" alt="Close Button"></a>
+      <ul class="padding-reset margin-reset">
+        <li><a href="#works">Portfolio</a></li>
+        <li><a href="#about-me">About</a></li>
+        <li><a href="#contact">Contact</a></li>
+  
+      </ul>
+    </nav>
+  </header>
+  <section id="headline">
+    <h1 id="heading" class="margin-reset">Hey There. <br>I'm    Sami ullah</h1>
+    <p class="sub-heading margin-reset">I’am a Software Developer</p>
```

Comment:
- Kindly change this `p` element to an `h2` or any other subheading as this allows search engines and screen readers to recognize that this is an important text. ✔